pyaedt/hfss.py

```diff
@@ -5406,6 +5407,8 @@ def set_differential_pair(
         """Add a differential pair definition.
 
         Differential pairs can be defined only in Terminal and Transient solution types.
+        The differential pair are created from an existing port definition having at least two
+        terminals in addition to a ground conductor.
```

Comment:
```suggestion
        The differential pair is created from an existing port definition having at least two
        terminals in addition to a ground conductor.
```
Pair is a singular noun referring to a group of objects. This means it takes a singular verb--in this case "is".

Reply:
Thanks. Done.